lucupy/minimodel/observation.py

```diff
@@ -153,7 +153,8 @@ class Observation:
     belongs_to: ProgramID
 
     # Some observations do not have constraints, e.g. GN-208A-FT-103-6.
-    constraints: Optional[Constraints]
+    # to mypy complae we should have an EmptyContraints
+    constraints: Constraints
```

Comment:
This is the one that has me a little concerned, because there are observations that don't have constraints. What do we do in that case? Was there a reason I'm missing why this had to be changed?